docs/getting_started.md

```diff
@@ -43,8 +43,18 @@ Rails.application.routes.draw do
  end
 ```
 
-The routes can be customized to show or hide
-different models on the dashboard.
+Removing resources from the routes will hide them on the dashboard.
+
+There might be situations where resources shall be hidden in the navigation
+but still be nested in another resource. In this case you keep the route but configure the dashboard to not be displayed in the navigation.
+
+```ruby
+class LineItemDashboard < Administrate::BaseDashboard
```

Comment:
What would you think of making this name more abstract or generic? Something like `RelationalRecordsDashboard` or `AssociationsDashboard` ? I found myself wondering if `LineItem` had any significance, and am curious to know if anyone else paused at that. 

Thanks for working towards an enhancement to `administrate`!